submissions/EmmanuelUdoh/HTMLTASK/login.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Kimaj: for Electronics, Phones, Computers, Fashion </title>
+        <link href="C:\Users\Emmanuel pc\Desktop\Dufuna-CodeCamp19\submissions\EmmanuelUdoh\HTMLTASK\style.css" rel="stylesheet" type="text/css">
```

Comment:
Hey @Eudoh940 you've done a great job. One change required though. Can you change this `link`'s `href` to a relative path instead of the absolute path you have used?

Simply make it 
```
<link href="style.css" rel="stylesheet" type="text/css">
```
Since you have the css file and html file in the same folder, it should work fine. Once this update is done, I think your code can be merged.